packages/atomic/src/components/insight/result-lists/atomic-insight-result-list/atomic-insight-result-list.tsx

```diff
@@ -73,6 +74,12 @@ export class AtomicInsightResultList
    */
   @Prop({reflect: true}) imageSize: ResultDisplayImageSize = 'icon';
 
+  /**
+   * Where to open the result link.
+   * @defaultValue `_blank`
+   */
+  @Prop() target: ResultTarget = '_blank';
```

Comment:
Instead of naming this `target`, I think it should include more context (e.g., that it only works in grid cells, and that it refers to the `target` property of anchor elements). Could be something like `gridCellLinkTarget`.

Also, I think the jsdoc above should:
* Include a link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target
* Specify that the property only works for `display="grid"`.